.github/workflows/seat_service_seatctrl_test.yml

```diff
@@ -50,7 +50,7 @@ jobs:
 
       - name: cobertura-report
         if: ${{ github.event.pull_request.head.repo.full_name == github.repository }}
-        uses: 5monkeys/cobertura-action@v13
+        uses: 5monkeys/cobertura-action@18d911b62098eafd8b9db93d6c74194556cc3515
```

Comment:
is there a reason we are using commit hash instead of a version here @int0x27 ? (If so, can we put a comment above why)

Reply:
They didn't have a released version, but a fix for node 12 deprecation warning here: https://github.com/5monkeys/cobertura-action/commit/18d911b62098eafd8b9db93d6c74194556cc3515
Also for security purposes it is better to pin the commit instead of using master

Reply:
It seems that the action is not that well maintained, which could give us problem in the long run. I agree that a comment would be beneficial.